rockcraft/oci.py

```diff
@@ -626,6 +626,22 @@ def _inject_architecture_variant(image_path: Path, variant: str) -> None:
     tl_index_path.write_bytes(json.dumps(tl_index).encode("utf-8"))
 
 
+def _run_skopeo(arguments: List[str], **kwargs: Any) -> subprocess.CompletedProcess:
+    """Find and run the correct 'skopeo' binary with ``arguments``."""
+    global _SKOPEO_PATH  # pylint: disable=global-statement
+    if _SKOPEO_PATH is None:
+        _SKOPEO_PATH = utils.get_snap_tool("skopeo")
```

Comment:
We know where we're installing the tools (e.g. `$SNAP/bin/skopeo`), can't we use that path directly?

Reply:
the intention of the methods is to run using the fully qualified paths in the snap and out of it for easier debugging (or log troubleshooting). How we resolve that I don't mind